lib/ansible/modules/cloud/azure/azure_rm_managed_disk.py

```diff
@@ -101,6 +106,14 @@
         resource_group: Testing
         disk_size_gb: 4
 
+    - name: Mount the managed disk to VM
```

Comment:
Is there a way to unmount a disk?

Reply:
Set the managed_by to None or Empty, I will add such example